book/src/keymap.md

```diff
@@ -320,10 +320,12 @@ Displays documentation for item under cursor. Remapping currently not supported.
 
 Displays documentation for the selected completion item. Remapping currently not supported.
 
-| Key                         | Description    |
-| ----                        | -----------    |
-| `Shift-Tab`, `Ctrl-p`, `Up` | Previous entry |
-| `Tab`, `Ctrl-n`, `Down`     | Next entry     |
+| Key                         | Description                      |
+| ----                        | -----------                      |
+| `Shift-Tab`, `Ctrl-p`, `Up` | Previous entry                   |
+| `Tab`, `Ctrl-n`, `Down`     | Next entry                       |
+| `Escape`                    | Close menu and accept completion |
```

Comment:
The behavior is a little nuanced for escape - it's not directly handled by completion. It's actually ignored by completion and anything ignored by completion (arrow keys for example) cause the completion to be accepted. We may want to remove this from the table here and then have a blurb below that says that anything other than what's in the table causes the completion item to be accepted.